jetson/science_bridge/src/__main__.py

```diff
@@ -0,0 +1,391 @@
+'''
+Writes, reads and parses NMEA like messages from the onboard
+science nucleo to operate the science boxes and get relevant data
+'''
+import serial
+import asyncio
+# import Adafruit_BBIO.UART as UART  # for beaglebone use only
```

Comment:
can this be like something set in a config file? 'if config.mode == testing import adafruit_bbio.uart' or something similar? same for the uart port?